현정/5주차_11047.java

```diff
@@ -0,0 +1,39 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.StringTokenizer;
+
+public class BJ_11047 {
+    static int n;
+    static int k;
```

Comment:
문제에서 n과 k로 준다고 하긴 했는데, 이러면 정확히 무슨 역할을 하는지 알기는 어려우니까
```suggestion
    static int kind;
    static int sum;
```
이런식으로 변수명을 작성하는게 더 좋아보이네유